test_runner/src/main/kotlin/ftl/run/platform/android/CreateAndroidTestContext.kt

```diff
@@ -50,13 +51,34 @@ private fun InstrumentationTestContext.calculateShards(
 
 private fun InstrumentationTestContext.getFlankTestMethods(
     testFilter: TestFilter
-): List<FlankTestMethod> =
-    DexParser.findTestMethods(test.local).asSequence().distinct().filter(testFilter.shouldRun).map { testMethod ->
-        FlankTestMethod(
-            testName = "class ${testMethod.testName}",
-            ignored = testMethod.annotations.any { it.name == "org.junit.Ignore" }
-        )
-    }.toList()
+): List<FlankTestMethod> {
+    val parameterizedClasses = mutableListOf<String>()
+    DexParser.readDexFiles(test.local).forEach { file ->
+        file.classDefs.forEach { classDef ->
+            val directory = file.getAnnotationsDirectory(classDef)
+            val className = file.formatClassName(classDef).dropLast(1)
+            val classAnnotations = file.getClassAnnotationValues(directory).map { it.name }
+            if (classAnnotations.any { it.toLowerCase().contains("RunWith".toLowerCase()) } && // sprawdzamy czy klasa ma adnotacje RunWith
+                file.getClassAnnotationValues(directory)
+                    .flatMap { annotations -> annotations.values.values }
+                    .filterIsInstance<DecodedValue.DecodedType>()
+                    .map { it.value }
+                    .any { it.toLowerCase().contains("Parameterized".toLowerCase()) }) {  // sprawdzamy czy klasa zadana jako parametr  adnotacji to Parameterized
```

Comment:
I wonder if there are any other ways to define tests that can be identified by `AndroidJUnitRunner` as parameterized